src/digitalmarketplace/all.js

```diff
@@ -1,10 +1,20 @@
+import './vendor/polyfills/NodeList/prototype/forEach'
 import CookieBanner from './components/cookie-banner/cookie-banner'
 import CookieSettings from './components/cookie-settings/cookie-settings'
 import * as Analytics from './components/analytics/analytics'
 import initAnalytics from './components/analytics/init'
 import { getConsentCookie } from './helpers/cookie/cookie-functions'
+import ListInput from './components/list-input/list-input'
 
 function initAll (options) {
+  // Set the options to an empty object by default if no options are passed.
+  options = typeof options !== 'undefined' ? options : {}
+
+  // Allow the user to initialise Digital Marketplace GOV.UK Frontend in only
+  // certain sections of the page
+  // Defaults to the entire document if nothing is set.
+  var scope = typeof options.scope !== 'undefined' ? options.scope : document
```

Comment:
Why is this needed? Asking more out of curiosity

Reply:
It looks like it was following the structure of govuk-frontend, so you can plug it in to certain bits of a page rather than the whole thing.

I think it's probably not needed for this particular repo, since we always use it on the whole page, but thought I'd leave it in to allow that functionality.

Can always revert if preferred - though as it stands we're passing in the options arg and not using it.